packages/hooks/src/hooks.spec.js

```diff
@@ -0,0 +1,62 @@
+import { act, cleanup, renderHook } from 'react-hooks-testing-library';
+import { useDocumentListener, useElementNode, useViewportSize } from './index';
+
+afterAll(cleanup);
+
+describe('packages/hooks', () => {
+  describe('useDocumentListener', () => {
+    test('event callback should fire when enabled is true', () => {
+      const eventCallback = jest.fn();
+
+      renderHook(() => useDocumentListener('click', eventCallback, true));
+
+      act(() => {
+        document.body.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+      });
+
+      expect(eventCallback).toHaveBeenCalledTimes(1);
+    });
+
+    test('event callback should not fire when enabled is false', () => {
+      const eventCallback = jest.fn();
+
+      renderHook(() => useDocumentListener('click', eventCallback, false));
+
+      act(() => {
+        document.body.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+      });
+
+      expect(eventCallback).toHaveBeenCalledTimes(0);
+    });
+  });
+
+  // describe('useMutationObserver', () => {})
+
+  describe('useElementNode', () => {
```

Comment:
Can make a simple function component to test that it gets a reference to the element and that the function is re-ran.